docs/configure-artifact-repository.md

```diff
@@ -107,43 +111,181 @@ $ cat > policy.json <<EOF
    ]
 }
 EOF
-$ aws s3 mb s3://$mybucket [--region xxx]
-$ aws iam create-user --user-name $mybucket-user
-$ aws iam put-user-policy --user-name $mybucket-user --policy-name $mybucket-policy --policy-document file://policy.json
-$ aws iam create-access-key --user-name $mybucket-user > access-key.json
 ```
 
 If you do not have Artifact Garbage Collection configured, you should remove `s3:DeleteObject` from the list of Actions above.
```

Comment:
You didn't change this part, but this should have a link. I'd also probably invert this and remove `s3:DeleteObject` initially and say to add it if you do have ArtifactGC enabled, i.e. following least privilege. Since you didn't change this, we can do that in a separate PR too.

```suggestion
If you do not have [Artifact Garbage Collection](walk-through/artifacts.md#artifact-garbage-collection) configured, you should remove `s3:DeleteObject` from the list of Actions above.
```